src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;

```

Comment:
Consider using access modifiers for your fields. If they are intended to be accessed only within this class or package, you should use 'private' to ensure encapsulation.